src/common/options/mds.yaml.in

```diff
@@ -287,15 +288,15 @@ options:
 - name: mds_sessionmap_keys_per_op
   type: uint
   level: advanced
-  desc: number of omap keys to read from the SessionMap in one operation
+  desc: Number of omap keys to read from the SessionMap in one operation
   default: 1_K
   services:
   - mds
   with_legacy: true
 - name: mds_recall_max_caps
-  type: size
+  type: uint
```

Comment:
This is the primary intent of this PR.  I noticed that a docs page that references this value reported 30000B as the default, which seems wrong for a number of caps, which should be a unitless integer.  Please validate and let me know if any concomitant update elsewhere within the codebase should be added to this PR.

Reply:
I think it's unfortunate that "size" implies a unit here. I chose `size` because it made it easier to specify a larger value like:

`ceph config set mds mds_recall_max_caps 100k`

if the main issue here is that `size` includes a unit, then perhaps it shouldn't. Alternatively, we could make `int`/`uint` accept SI prefixes.

Reply:
Pushing an update.